### DISCUSSION TITLE
Is it possible to move the floating tree to the right?

### DISCUSSION THREAD

**Discussion by nyngwang** [nvim-neo-tree/neo-tree.nvim#88] [Q&A] [answered]
As title 🥺

**Comment by cseickel** [comment, marked as answer]
Yup, here's an example, it's actually the config that I use for floating:
https://github.com/nvim-neo-tree/neo-tree.nvim/wiki/Visual-Customizations#float-right

**Comment by nyngwang** [comment]
@cseickel: You guys are so awesome! I just decided to migrate from nvim-tree.lua to here :)
